mmhuman3d/core/conventions/keypoints_mapping/openpose.py

```diff
@@ -166,3 +166,124 @@
     'right_smalltoe_openpose',
     'right_heel_openpose'
 ]
+
+OPENPOSE_118_KEYPOINTS = [
```

Comment:
It would be very helpful to add comments about the differences amongst openpose_{25, 118, 135}.

Reply:
Add descriptions about the differences in https://github.com/open-mmlab/mmhuman3d/pull/102/commits/7fb3d11ce45b1574d89310c138db6341607e8e19